docs/en/advanced/Alluxio-Storage-Management.md

```diff
@@ -12,22 +12,50 @@ priority: 0
 
 ## Alluxio Storage Overview
 
-TODO(Zac): Add an overview of Alluxio storage here, covering
+
+<!-- ################## -->
+<!-- TODO(Zac): Add an overview of Alluxio storage here, covering
 - relationship and difference between Alluxio storage and under storage.
     * replication: hot data in Alluxio storage can have mulitple copies
     * once data is persisted, data can have zero copy in Alluxio but still available to
    applications.
-- the indication to application I/O performance when data is cached in Alluxio storage.
-
-Alluxio manages the local storage, including memory, of Alluxio workers to act as a distributed
-buffer cache. This fast data layer between user applications and the various under storages results
-in vastly improved I/O performance.
+- the indication to application I/O performance when data is cached in Alluxio storage. -->
+<!-- ################## -->
+
+Alluxio helps unify users' data across a variety of platforms while also helping to increase
```

Comment:
nit: I would emphasize less on the unification side in this doc to avoid confuse readers